featuremanagement/opentelemetry/_send_telemetry.py

```diff
@@ -0,0 +1,92 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from logging import getLogger as get_logger, INFO
+from .._models import VariantAssignmentReason
+
+
+_event_logger = get_logger(__name__)
+
+try:
+    from opentelemetry.sdk._logs import LoggingHandler
+
+    HAS_OPENTELEMETRY_SDK = True
+except ImportError:
+    HAS_OPENTELEMETRY_SDK = False
+    _event_logger.warning("opentelemetry-sdk is not installed. Telemetry will not be sent to Open Telemetry.")
+
+try:
+    from azure.monitor.events.extension import track_event as azure_monitor_track_event
+
+    HAS_AZURE_MONITOR_EVENTS_EXTENSION = True
+except ImportError:
+    HAS_AZURE_MONITOR_EVENTS_EXTENSION = False
+
+FEATURE_NAME = "FeatureName"
+ENABLED = "Enabled"
+TARGETING_ID = "TargetingId"
+VARIANT = "Variant"
+REASON = "VariantAssignmentReason"
+
+EVENT_NAME = "FeatureEvaluation"
+
+_event_logger.propagate = False
+
+
+class _FeatureManagementEventsExtension:
+    _initialized = False
+
+    @staticmethod
+    def initialize():
+        """
+        Initializes the logger to use an OpenTelemetry logging handler, if not already initialized.
+        """
+        if not _FeatureManagementEventsExtension._initialized:
+            _event_logger.addHandler(LoggingHandler())
+            _event_logger.setLevel(INFO)
+            _FeatureManagementEventsExtension._initialized = True
+
+
+def track_event(event_name, user, event_properties=None):
+    """
+    Track an event with the specified name and properties.
+
+    :param str event_name: The name of the event.
+    :param str user: The user ID to associate with the event.
+    :param dict[str, str] event_properties: A dictionary of named string properties.
+    """
+    if not HAS_OPENTELEMETRY_SDK:
+        return
+    if event_properties is None:
+        event_properties = {}
+    if user:
+        event_properties[TARGETING_ID] = user
+    if HAS_AZURE_MONITOR_EVENTS_EXTENSION:
+        azure_monitor_track_event(event_name, event_properties)
+        return
```

Comment:
I don't understand why if HAS_AZURE_MONITOR_EVENTS_EXTENSION, _event_logger will be skipped?